cypress/support/fragments/settings/remote-storage/remoteStorage.js

```diff
@@ -0,0 +1,15 @@
+import { NavListItem, Pane } from '../../../../../interactors';
+
+const remoteStorageNavigationPane = Pane('Remote storage');
+const configurationsItem = NavListItem('Configurations');
+const accessionTablesItem = NavListItem('Accession tables');
+
+export default {
+  checkSettingItems() {
+    cy.expect(remoteStorageNavigationPane.find(configurationsItem).exists());
+    cy.expect(remoteStorageNavigationPane.find(accessionTablesItem).exists());
```

Comment:
if multiple assertions in a row, better to pass as an array

Reply:
fixed